scripts/scripts.js

```diff
@@ -107,15 +106,6 @@ function decorateAnchors(element = document) {
   if (discoverMoreAnchors.length > 0) {
     decorateDiscoverMore(discoverMoreAnchors);
   }
-
-  const socialShareAnchors = anchors.filter((anchor) => {
-    const img = anchor.querySelector('img');
-    return img && img.src.includes('/icons/gray-share-icon.svg');
-  });
-
-  if (socialShareAnchors.length > 0) {
-    decorateSocialShare(socialShareAnchors);
-  }
```

Comment:
Is this no longer needed? Can't find it moved to anywhere else.

Reply:
@sdmcraft  correct, in eager phase we do not have social button. and buttons loaded in lazy phase are decorated already